### DISCUSSION TITLE
Make `description` field optional for Highlghted Cards

### DISCUSSION THREAD

**Discussion by carlosgarciaibanez** [Telefonica/mistica-design#1657] [Proposals]
### Goal

We want to simplify the management of this type of components, it doesn't seem to make much sense to have a mandatory description in a component like this.

### Context

We are currently using the Highlighted Cards for modules that do not require a description, like our Airtime Rewards module:
![image](https://github.com/Telefonica/mistica-design/assets/4517448/4ee577af-8358-486d-9f0d-ab386e8419e9)
To achieve this, we are currently setting the _description_ field to a blank space, but this is a brittle and problematic solution.
The proper way to go would be to make the field completely optional.

### Impact

This will simplify things and will allow us to remove some dirty hacks.

### Discussion Questions

- Why was the reason to make the _description_ field mandatory in this component?
- If that is a compelling reason, why did we allow these modules without a description on them?

**Comment by yceballost** [comment]
We agree to include description as optional prop!
